app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -225,15 +225,15 @@ class AdministratorControlsActivityTest {
       onView(
         atPositionOnView(
           R.id.administrator_controls_list,
-          2,
+          position = 2,
           R.id.topic_update_on_wifi_switch
```

Comment:
We want named arguments for all 4 parameters.

Reply:
Have made named arguments out of the parameters, as you mentioned,